application/world/services/auto_bible_generator.py

```
@@ -261,17 +261,88 @@ async def generate_and_save(
+        # 1. 提炼搜索关键词 (不使用 JSON 以求稳定)
+        extract_prompt = Prompt(
+            system="你是资料检索专家。请从用户的创意中提取最核心的 2 个背景搜索词（如具体年代、地域、行业、或者特定历史事件）。\n【严格约束】绝对不要输出 JSON！只需用逗号分隔两个词语即可，例如：1990年深圳物价, 华强北BB机倒卖",
+            user=f"创意：{premise}\n请输出2个最需要考据的搜索关键词："
+        )
+        config = GenerationConfig(model=target_model, max_tokens=100, temperature=0.3)
+        keyword_result = await actual_llm_service.generate(extract_prompt, config)
+        
+        # 强制清理可能的 JSON 残留
+        content = keyword_result.content.strip()
+        if "{" in content or "[" in content:
+            keywords = ["1990年深圳", "华强北 倒卖"] # 如果大模型不听话硬回 JSON，强行写死兜底，避免查出一堆乱码
```

Ruff (0.15.10)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)